model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -0,0 +1,188 @@
+        self._rcg = None
+
+    def set_last_results(self, measurements):
+        self._rcg.set_last_results(measurements)
+
+    def get_configs(self) -> Generator[RunConfig, None, None]:
+        """
+        Returns
+        -------
+        RunConfig
+            The next RunConfig generated by this class
+        """
+        yield from self._execute_quick_search()
+        yield from self._sweep_concurrency_over_top_results()
+
+    def _execute_quick_search(self):
+        self._rcg = self._create_quick_run_config_generator()
+
+        yield from self._rcg.get_configs()
+
+    def _create_quick_run_config_generator(self) -> QuickRunConfigGenerator:
+        return QuickRunConfigGenerator(
+            search_config=self._search_config,
+            config=self._config,
+            gpus=self._gpus,
+            models=self._models,
+            client=self._client,
+            model_variant_name_manager=self._model_variant_name_manager)
+
+    def _sweep_concurrency_over_top_results(self):
+        top_results = self._result_manager.top_n_results(
+            n=DEFAULT_NUM_CONFIGS_PER_MODEL)
+
+        for count, result in enumerate(top_results):
+            new_config = self._create_new_config_command_profile(result)
+            self._rcg = self._create_brute_run_config_generator(
+                new_config, skip_default_config=(count != 0))
+
+            yield from self._rcg.get_configs()
+
+    def _create_new_config_command_profile(
+            self, result: RunConfigResult) -> ConfigCommandProfile:
+        new_config = deepcopy(self._config)
+
+        self._set_search_mode(new_config)
+        self._set_parameters(result, new_config)
+
+        return new_config
+
+    def _create_brute_run_config_generator(
+            self, new_config: ConfigCommandProfile,
+            skip_default_config: bool) -> BruteRunConfigGenerator:
+        return BruteRunConfigGenerator(
+            config=new_config,
+            gpus=self._gpus,
+            models=self._models,
+            client=self._client,
+            model_variant_name_manager=self._model_variant_name_manager,
+            skip_default_config=skip_default_config)
+
+    def _set_search_mode(self, config: ConfigCommandProfile):
+        config._fields['run_config_search_mode']._field_type._value = 'brute'
+        config._fields['run_config_search_disable']._field_type._value = False
+        config._fields['early_exit_enable']._field_type._value = True
+
+    def _set_parameters(self, result: RunConfigResult,
+                        config: ConfigCommandProfile):
+        batch_size = self._find_batch_size(result)
+        self._set_batch_size(config, batch_size)
+
+        instance_count = self._find_instance_count(result)
+        self._set_instance_count(config, instance_count)
+
+        self._set_concurrency(config)
+
+    def _find_batch_size(self, result: RunConfigResult) -> int:
+        return result.run_config().model_run_configs()[0].model_config(
```

Comment:
hardcoded 0? Will this break for multi-model?

Reply:
Yes, this is only going to work for single model, but didn't think this was an issue (at the moment), since quick search is only for single model.

Reply:
For multi-model, this approach isn't going to work as we can't use the search space to set batch size or instance count if each model has a different value.

Reply:
It seems we picked the wrong approach then. We can keep this for now, but create a story to convert to a brute force config file in the multi-model epic, and put a TODO tag for these [0] lines